src/components/controls-shape-offset.js

```diff
@@ -34,8 +36,8 @@ AFRAME.registerComponent("controls-shape-offset", {
       quaternion.conjugate();
 
       const shape = {
-        shape: "sphere",
-        radius: "0.02",
+        shape: "box",
+        halfExtents: { x: 0.03, y: 0.04, z: 0.05 },
```

Comment:
Did a box end up feeling better than the sphere? Or is change better for perf (I thought sphere collision checks were fast)?

Reply:
box felt better to me

Reply:
not a change for perf, but to better capture the sense that putting an object near your palm will grab it